Resources/Prototypes/Corvax/Recipes/Reactions/drinks.yml

```
@@ -0,0 +1,171 @@
+- type: reaction
+  id: Yorsh
+  reactants:
+    Beer:
+      amount: 4
+    Vodka:
+      amount: 1
+  products:
+    Yorsh: 5
+
+- type: reaction
+  id: Alexandr
+  reactants:
+    Cognac:
+      amount: 1
+    Coffee:
+      amount: 1
+    Cream:
+      amount: 1
+  products:
+    Alexandr: 3
+
+- type: reaction
+  id: Daiquiri
+  reactants:
+    Rum:
+      amount: 4
+    JuiceLime:
+      amount: 1
+    Sugar:
+      amount: 1
+  products:
+    Daiquiri: 6
+
+- type: reaction
+  id: MojitoAlt
+  reactants:
+    Daiquiri:
+      amount: 6
+    JuiceLime:
+      amount: 3
+    SodaWater:
+      amount: 4
+    Sugar:
+      amount: 3
+  products:
+    Mojito: 16
```

Бармены с ума сойдут, имхо, лучше:
```suggestion
- type: reaction
  id: MojitoAlt
  reactants:
    Daiquiri:
      amount: 1
    JuiceLime:
      amount: 1
    SodaWater:
      amount: 1
    Sugar:
      amount: 1
  products:
    Mojito: 4
```

---

На самом деле нет. Если цель скрафтить мохито, то ему потребуется также прокликать только 4 реагента, как и раньше. К примеру, возьмем по 5 единиц.
Если он случайно сделает дайкири, то будет
- лаймовый сок 3.75
- сахар 3.75
- дайкири 7.5
Тогда, при добавлении 5-ти газированной воды, у него получится 20 мохито